Examples/Python/src/Geometry.cpp

```diff
@@ -275,36 +276,61 @@ void addExperimentalGeometry(Context& ctx) {
   }
 
   {
-    using Range2D = Acts::RangeXD<2u, Acts::ActsScalar>;
-    using KdtSurfaces2D = Acts::Experimental::KdtSurfaces<2u, 100u>;
-    using KdtSurfacesProvider2D =
+    using RangeXDDim1 = Acts::RangeXD<1u, Acts::ActsScalar>;
+    using KdtSurfacesDim1Bin100 = Acts::Experimental::KdtSurfaces<1u, 100u>;
+    using KdtSurfacesProviderDim1Bin100 =
+        Acts::Experimental::KdtSurfacesProvider<1u, 100u>;
```

Comment:
I guess 100 just means "big enough" here?

might be worth putting a comment

Reply:
Makes sense, will do.